examples/plot.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Create plots for privacy evaluation of "shuffling+flipping"."""
+
+from buffling import snr
+import numpy as np
+import pandas as pd
+import seaborn as sns 
+import matplotlib.pyplot as plt
+
+## global parameters
+NUM_PUBS = 4
+FLIP_PROB = 0.05
+# plt.style.use("ggplot")
+
+## dependent parameters
+NUM_LOGO = 2 ** NUM_PUBS
+TRANSIT_MAT = snr.generate_transition_matrix(NUM_PUBS, FLIP_PROB)
+
+
+def main():
+
+  ## heatmap: SNR vs (i, j)
+  fig, ax = plt.subplots(1,4,figsize=(16, 3))
+  for i in range(4): 
+    skewness = 3 * i 
+    input_logo_wgt = np.exp(- skewness * (np.arange(NUM_LOGO) + 1) / NUM_LOGO)
+    input_logo_wgt /= sum(input_logo_wgt)
+    s = snr.calculate_signal_to_noise_ratio(input_logo_wgt, TRANSIT_MAT)
+    sns.heatmap(np.sqrt(s), ax = ax[i])
+    ax[i].set_title(f"SNR (skewness: {skewness})")
+  plt.savefig('docs/fig/1_snr_vs_ij.pdf')
+  plt.close()
+
+
```

Comment:
There is no need to put two empty lines here. One is enough.

Reply:
Deleted one empty line here.